pkg/core/mempool/mempool.go

```diff
@@ -494,6 +491,29 @@ func (m *Mempool) kadcastTx(t TxDesc) error {
 	return nil
 }
 
+// RequestUpdates sends topics.MemPool to N Kadcast Network nodes.
+func (m *Mempool) RequestUpdates() {
+	maxNodes := config.
+		Get().
+		Mempool.
+		MaxNumUpdaters
+
+	if maxNodes == 0 {
+		log.Warn("updates disabled")
```

Comment:
Ok for the warning. But I would set a default value greater than 0

Reply:
Valid point.